src/main/java/org/broadinstitute/hellbender/utils/codecs/table/TableFeature.java

```diff
@@ -70,14 +70,6 @@ public Locatable getLocation() {
         return this.position;
     }
 
-    public List<String> getAllValues() {
-        return getValuesTo(values.size());
-    }
-
-    public List<String> getValuesTo(int columnPosition) {
-        return values.subList(0, columnPosition);
-    }
```

Comment:
We agreed during the original codec code review that these methods were essential for making `TableFeature` actually usable by clients and should not be deleted (even if currently unused).